pipprivates/install.py

```diff
@@ -36,10 +36,11 @@ def collect_requirements(fname, transform_with_token=None):
         # an oauth key
         elif tokens[0] == '-e':
             if tokens[1].startswith('git+git@github.com:'):
-                vcs_url = 'git+https://{}:x-oauth-basic@github.com/{}'.format(
-                    transform_with_token, tokens[1][19:]) if transform_with_token else tokens[1]
-
-                collected += [vcs_url]
+                if transform_with_token:
+                    collected.append('git+https://{}:x-oauth-basic@github.com/{}'.format(
+                        transform_with_token, tokens[1][19:]))
+                else:
+                    collected.append('-e {}'.format(tokens[1]))
```

Comment:
Why the `-e`, shouldn't that be absent in this context?. The previous oneliner didn't contain the `-e` option.